grc/gui/canvas/param.py

```diff
@@ -156,6 +160,21 @@ def format_block_surface_markup(self):
         Returns:
             a pango markup string
         """
+
+        if str(self.value) == str(self._evaluated): # This is not a parameter defined by a variable
```

Comment:
I believe `value` is already a `str`
Also, please update comment to wording change mentioned above, e.g. a value of "1+1" will result in false even though it not defined by a variable

Reply:
You're right, this is probably not the best test to achieve the goal stated in the comment.  This also applies to things that are variables in other classes (ie, the wfm in the signal source block or the constellations in the demod block)

Do you think there's a cleaner way to determine if the string the user inputs is an original input vs something that's evaluated?